packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -95,226 +96,232 @@ const getTitle = (market_type: string, show_eu_related_content: boolean) => {
     return market_type;
 };
 
-const DMT5TradeModal = ({
-    mt5_trade_account,
-    show_eu_related_content,
-    onPasswordManager,
-    toggleModal,
-}: TMT5TradeModalProps) => {
-    const getCompanyShortcode = () => {
-        if (
-            (mt5_trade_account.account_type === 'demo' &&
-                mt5_trade_account.market_type === 'financial' &&
-                mt5_trade_account.landing_company_short === 'labuan') ||
-            mt5_trade_account.account_type === 'real'
-        ) {
-            return mt5_trade_account.landing_company_short;
-        }
-        return undefined;
-    };
+const DMT5TradeModal = observer(
+    ({ mt5_trade_account, show_eu_related_content, onPasswordManager, toggleModal }: TMT5TradeModalProps) => {
+        const { ui } = useStore();
+        const { is_mobile } = ui;
```

Comment:
```suggestion
        const { is_mobile } = ui;
```
Can we not use this and just hide it via css?

Reply:
done